src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contract in the address book.
+ */
+
+public class Contact {
+	
+    public final String value;
```

Comment:
try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.